### PR TITLE
issue_112_period

### PR DESCRIPTION
**Pull request by pdeiml**
I think the period is not used in the output files, hence, checking the unit transformation by astropy could not be checked.

python make.py all works without errors.

Ready to merge

### CODE REVIEW

**Comment by pdeiml** [comment]
Do I understand you right? You want e.g. that 'glon' is like this:

      glon:
        type: object
       unit: deg
        additionalProperties: false
        description: |
          Galactic longitude 
          A quantity string with unit equal to 'degrees'.
        properties:
          val: {type: string, description: Measured value}
          err: {type: string, description: Statistical error}
          err_sys: {type: string, description: Systematic error}

So that everywhere 'unit: xxx' is mentioned?

**Comment by cdeil** [comment]
Suggest to put this:
```
 glon:
    type: object
    additionalProperties: false
    description: |
      Galactic longitude 
      Properties are quantity strings with unit equivalent to "deg"
    properties:
      val: {type: string, description: Measured value}
      err: {type: string, description: Statistical error}
      err_sys: {type: string, description: Systematic error}
```

**Comment by pdeiml** [comment]
Now, everything should be fine.
RTM

**Comment by pdeiml** [comment]
Solved the conflicts, rebased against master.

RTM

File: input/schemas/dataset_source_info.schema.yaml [hunk 1]
```diff
@@ -167,14 +167,16 @@ properties:
       period:
         type: object
         additionalProperties: false
-        unit: day
-        description: Orbital period
+        description: |
+          Orbital period
+          Property 'unit' is a string that can be parsed by Astropy Units.
```
Comment (cdeil) [inline input/schemas/dataset_source_info.schema.yaml]: Suggest to remove this here:
```
Property 'unit' is a string that can be parsed by Astropy Units.
```
and to put this info below:
```
unit: {type: string, description: Unit equivalent with "day".}
```

**Comment by pdeiml** [comment]
Changed it, 

RTM

**Comment by cdeil** [comment]
Thanks!

**Pull request merged**